compiler.cpp

```diff
@@ -286,6 +286,7 @@ const Strat* ComputeStrategy(const Policy& node, std::unordered_map<const Policy
         }
         case Policy::Type::OR: {
             if (node.sub.size() != 2) return {};
+            if (node.prob[0] + node.prob[1] == 0) return {};
```

Comment:
This is not enough; you can have two probabilities that each individually are below the limti, but summed together cause an overflow (but above 0).

One trick is testing whether (a + b < a).

Reply:
Good point. For correctness reasons we should guard not only against the division by zero case, but for any wraparound. Now fixed. Please re-review :-)